Bottom: a08e539e76bc44c571aa1e354db82dd36a5a72ac
Top:    1324d56688023968aa7f4e3f852a9482467e62c5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-07 11:34:39 -0500

Refresh of add-docker-option

---
diff --git a/ansible-playbook/nexus.yml b/ansible-playbook/nexus.yml
deleted file mode 100644
index 5030618a..00000000
--- a/ansible-playbook/nexus.yml
+++ /dev/null
@@ -1,13 +0,0 @@
----
-    
-- name: Install Sonatype Nexus Repository Manager 3
-  hosts: daedalus
-  become: true
-  become_user: root
-  become_method: sudo
-  tasks:
-  - set_fact:
-      openssl_dv_name: "openssl_DV"
-  - name: Run Nexus role
-    include_role:
-      name: nexus3
